content/applications/finance/accounting/fiscal_localizations/localizations/colombia.rst

```diff
@@ -1,276 +1,41 @@
-should be configured and synchronized with the CEN Financiero.
-
-.. image:: colombia/colombia10.png
-   :align: center
-
-
-Users
-~~~~~
-
-The default template that is used by Odoo on the invoice PDF includes
-the job position of the salesperson, so these fields should be
-configured:
-
-.. image:: colombia/colombia11.png
-   :align: center
-
-
-Usage and testing
-=================
-
-Invoice
--------
-
-When all your master data and credentials has been configured, it's
-possible to start testing the electronic invoice workflow.
-
-
-Invoice creation
-~~~~~~~~~~~~~~~~
-
-The functional workflow that takes place before an invoice validation
-doesn't change. The main changes that are introduced with the
-electronic invoice are the next fields:
-
-.. image:: colombia/colombia12.png
-   :align: center
-
-There are three types of documents:
-
-- **Factura Electronica**: This is the regular type of document and
-  its applicable for Invoices, Credit Notes and Debit Notes.
-- **Factura de Importación**: This should be selected for importation
-  transactions.
-- **Factura de contingencia**: This is an exceptional type that is
-  used as a manual backup in case that the company is not able to use
-  the ERP and it's necessary to generate the invoice manually, when
-  this invoice is added to the ERP, this invoice type should be
-  selected.
-
-
-Invoice validation
-~~~~~~~~~~~~~~~~~~
-
-After the invoice is validated an XML file is created and sent
-automatically to Carvajal, this file is displayed in the chatter.
-
-.. image:: colombia/colombia13.png
-   :align: center
-
-An extra field is now displayed in "Other Info" tab with the name of
-the XML file. Additionally there is a second extra field that is
-displayed with the Electronic Invoice status, with the initial value
-"In progress":
-
-.. image:: colombia/colombia14.png
-   :align: center
-
-
-Reception of legal XML and PDF
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-The electronic invoice vendor receives the XML file and proceeds to
-validate the structure and the information in it, if everything is
-correct the invoice status changes to "Validated" after using the
-"Check Carvajal Status" button in the Action dropdown. They then
-proceed to generate a Legal XML which includes a digital signature and
-a unique code (CUFE), a PDF invoice that includes a QR code and the
-CUFE is also generated.
-
-After this:
-
-- A ZIP containing the legal XML and the PDF is downloaded and
-  displayed in the invoice chatter:
-
-  .. image:: colombia/colombia15.png
-
-  .. image:: colombia/colombia16.png
-
-- The Electronic Invoice status changes to "Accepted"
-
-
-Common errors
-~~~~~~~~~~~~~
-
-During the XML validation the most common errors are usually related
-to missing master data. In such cases, error messages are shown in the
-chatter after updating the electronic invoice status.
+Electronic invoicing for Colombia is available for Odoo 12 and above, and requires the following
```

Comment:
@samueljlieber we got the advise to avoid mentioning odoo versions as the idea is to reuse the same documentation across multiple version whenever is aplicable.  

Probably we can replace this sentece by: "Now you can generate Electronic Invoice in Odoo and get the Government athorization using the next Colombian Localization modules"

What do you think?

cc. @vbe-odoo @fvz-odoo

Reply:
Thanks @ren-odoo that is a good suggestion, I'd just modify it a little to avoid using "you":
```suggestion
Electronic Invoices in Odoo can now be generated to get Government authorization by using the following Colombian Localization modules. 
```